src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -179,22 +380,75 @@
      */
     public static final Key<Value<BrickType>> BRICK_TYPE = KeyFactory.fake("BRICK_TYPE");
 
+    /**
+     * Represents the {@link Key} for whether an {@link Entity} can breed.
+     *
+     * @see BreedableData#breedable()
+     */
     public static final Key<Value<Boolean>> CAN_BREED = KeyFactory.fake("CAN_BREED");
 
+    /**
+     * Represents the {@link Key} for whether a [@link {@link FallingBlock} can
+     * drop as an item.
+     *
+     * @see FallingBlockData#canDropAsItem()
+     */
     public static final Key<Value<Boolean>> CAN_DROP_AS_ITEM = KeyFactory.fake("CAN_DROP_AS_ITEM");
 
+    /**
+     * Represents the {@link Key} for whether a {@link Humanoid} can fly.
+     * <p>For a {@link Player} this means he is able to toggle flight mode by
+     * double-tapping his jump button.</p>
+     *
+     * @see FlyingAbilityData#canFly()
+     */
     public static final Key<Value<Boolean>> CAN_FLY = KeyFactory.fake("CAN_FLY");
 
+    /**
+     * Represents the {@link Key} for whether a {@link Living} entity. This
+     * mostly applies to {@link Enderman} or {@link Creeper}s, but also to
+     * some projectiles like {@link Fireball}s.
+     *
+     * @see GriefingData#canGrief()
+     */
     public static final Key<Value<Boolean>> CAN_GRIEF = KeyFactory.fake("CAN_GRIEF");
 
+    /**
+     * Represents the {@link Key} for whether a {@link FallingBlock} will place
+     * itself upon landing.
+     *
+     * @see FallingBlockData#canPlaceAsBlock()
+     */
     public static final Key<Value<Boolean>> CAN_PLACE_AS_BLOCK = KeyFactory.fake("CAN_PLACE_AS_BLOCK");
 
+    /**
+     * Represents the {@link Key} for the {@link Villager}'s {@link Career}.
+     *
+     * @see CareerData#type()
+     */
     public static final Key<Value<Career>> CAREER = KeyFactory.fake("CAREER");
 
+    /**
+     * Represents the {@link Key} for the rotation of the
+     * {@link BodyParts#CHEST}.
+     *
+     * @see BodyPartRotationalData#bodyRotation()
+     */
     public static final Key<Value<Vector3d>> CHEST_ROTATION = KeyFactory.fake("CHEST_ROTATION");
 
+    /**
+     * Represents the {@link Key} for the {@link CoalType} of an item.
```

Comment:
I wasnt really referring to an Item entity, but I see how it's confusing. Will clarify.